docs/pages/hooks/useCreateNFT.mdx

```diff
@@ -20,14 +20,15 @@ export default function Component() {
 
   const handleClick = async () => {
     await createNFT({
```

Comment:
From an API point of view, this function creates an Asset so it should be named `createAsset`
On one side I would like to have the same name everywhere, on the other side, NFT is must more explicit. Maybe this would need to be rename on the backend in a future version?

Reply:
Yes, this one is indeed confusing, it was initially to compensate for the fact that the asset was used for everything (including erc20), but now that the API considers assets only nft, it might make sense actually to either use nft everywhere or asset everywhere, but this mix is not nice indeed.